html/semantics/scripting-1/the-script-element/script-type-whitespace.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<title>&lt;script type> non-ASCII whitespace handling</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+function testParserInsertedDidNotRun(description) {
+  test(() => assert_false(window.ran),
+       "Script shouldn't run with " + description + " (parser-inserted)");
+  window.ran = false;
```

Comment:
This should be in a t.add_cleanup so it executed even if the assert fails

Reply:
It's not encapsulated in `test()` though and matches how this is used in sibling files. I think this works out.